rotkehlchen/tasks/manager.py

```diff
@@ -380,6 +384,24 @@ def _maybe_decode_evm_transactions(self) -> None:
                 tx_hashes=hashes,
             )
 
+    def _check_premium_status(self) -> None:
+        """
+        Validates the premium status of the account and if the credentials are not valid
+        it logouts the user.
+        """
+        db_credentials = self.database.get_rotkehlchen_premium()
```

Comment:
This is going to be running almost every time the task manager is called which is way too often.

Make this check run in a configurable time period. Once every hour by default?